src/train_model.py

```diff
@@ -169,13 +170,26 @@ def train_model(config: Struct):
             plugins=[SLURMEnvironment(requeue_signal=signal.SIGHUP)],
             callbacks=[early_stopping, model_checkpoint],
             logger=tb_logger)
-
+        
+    ## Set up carbon emissions tracker
+        
+    CO2_csv_path = config.CO2_outfile if config.CO2_outfile else None
+
+    emissions_tracker = OfflineEmissionsTracker(
+                output_dir=model_dir,
+                output_file=CO2_csv_path,
+                country_iso_code="USA",
+                cloud_provider="gcp",
```

Comment:
Why is the cloud provider Google Cloud? We are using an offline supercomputer.

Reply:
If no provider is specified, then the country region's energy consumption profile is used (e.g., 65% coal, 15% hydro, 20% solar, whatever). If no area is specified, then the country's default average is used.  Due to the uniquely high demand, BYU built its own cleaner-than-local-standard plant to produce ~30% of its total energy needs for it's facilities. Thus, BYU' consumption profile deviates from the region's profile. I found Google's SLC datacenter profile to be pretty comparable to BYUs own, as I understand it.